src/librustdoc/html/format.rs

```diff
@@ -412,6 +412,20 @@ impl clean::GenericBound {
                 })?;
                 ty.print(cx).fmt(f)
             }
+            clean::GenericBound::Use(args) => {
+                if f.alternate() {
+                    f.write_str("use&lt;")?;
+                } else {
+                    f.write_str("use<")?;
+                }
```

Comment:
This is the wrong way around.

> `<` only if `f.alternate()` otherwise `&lt;`

With that I meant `if alternate { < } else { &lt; }` (pseudo)

Reply:
So by default, we escape. I think the tests don't catch this because their capture groups either start with `<'a` or are `<>`, none of which are valid HTML tags unlike e.g. `use<T>`, so they don't get turned into ones.

Reply:
i have no idea why i didn't get this right 🤦